cli_snapshots/build.rs

```diff
@@ -9,16 +9,26 @@ fn main() {
   let c = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
   let o = PathBuf::from(env::var_os("OUT_DIR").unwrap());
   let js_dir = c.join("../js");
+  let import_maps = vec![
+    deno_ops_fs_bundle::get_import_map(),
+    deno_bundle_util::get_import_map(),
+    deno_dispatch_json_bundle::get_import_map(),
```

Comment:
I don't really understand the purpose/necessity of this import map stuff.

Reply:
It should't be necessary for deno cli, but I'm trying to keep deno core embedding in mind here. 
No matter where `deno_ops_fs_bundle` is in relation to the project it's being used in, if I include the import maps `deno_ops_fs` will always resolve to the absolute path to the library entry point.